CSharp/AccountManagement/Program.cs

```diff
@@ -207,21 +221,21 @@ private static string PromptUserForAccountName()
             Console.Write("Enter the name of the Batch account to create: ");
             string accountName = Console.ReadLine();
             Console.WriteLine();
-
+            
             return accountName;
         }
 
         /// <summary>
         /// Creates a resource group. The user's Batch account will be created under this resource group.
         /// </summary>
-        /// <param name="resourceManagementClient">The <see cref="Microsoft.Azure.Management.Resources.IResourceManagementClient"/> 
+        /// <param name="resourceManagementClient">The <see cref="Microsoft.Azure.Management.ResourceManager.IResouceManagementClient"/>
         /// to use when creating the resource group.</param>
         /// <param name="location">The location where the resource group will be created.</param>
         /// <returns>A <see cref="System.Threading.Tasks.Task"/> object that represents the asynchronous operation.</returns>
         private static async Task CreateResourceGroupAsync(IResourceManagementClient resourceManagementClient, string location)
         {
-            ResourceGroupExistsResult existsResult = await resourceManagementClient.ResourceGroups.CheckExistenceAsync(ResourceGroupName);
-            if (!existsResult.Exists)
+            bool? existsResult = await resourceManagementClient.ResourceGroups.CheckExistenceAsync(ResourceGroupName);
+            if (!existsResult.Value)

```

Comment:
Should probably check for null before accessing the .Value property.  It's a little odd to me that a 3rd state exists at all - either the group exists or it doesn't, so what does the third state even mean?  But yeah, the check should probably be "if (existsResult == null || !existsResult.Value)"